### PR TITLE
🙈 Removed mix-manifest.json

### PR DESCRIPTION
**Pull request by MrKrisKrisu**

### CODE REVIEW

**Comment by MrKrisKrisu** [comment]
Ah, lol. I'm stupid.

**Comment by MrKrisKrisu** [comment]
Hmm, now the tests are failing, because the pre-generated manifest isn't there... But I don't really want to add `npm run` to the PHPUnit test...

File: .gitignore [hunk 1]
```diff
@@ -3,6 +3,7 @@ node_modules/
 npm-debug.log
 report/
 clover.xml
+public/mix-manifest.json
```
Comment (HerrLevin) [inline .gitignore:7]: shouldn't there also be a `mix-manifest.json`?

**Review by HerrLevin — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (MrKrisKrisu) [reply .gitignore:7]: Yes, there should be a `mix-manifest.json`. But not in the repo as it is regenerated everytime `npm run x` is running.

**Review by HerrLevin — APPROVED** [review]

**Pull request closed without merging**